Ash.FrontEnd/src/js/global/components/Fido.jsx

```diff
@@ -149,8 +160,15 @@ class Fido extends React.Component {
    */
   onCall = key => (configOverride = {}) => {
     const fullConfigs = this.getFullConfigs();
-    const newFullConfig = { ...fullConfigs[key], ...configOverride };
-    this.dispatchFetch(key, newFullConfig);
+
+    if (fullConfigs[key]) {
+      const newFullConfig = { ...fullConfigs[key], ...configOverride };
+      this.dispatchFetch(key, newFullConfig);
+    }
+
+    throw new Error(
+      `Config for "${key}" fetch is no longer in props. Very sad.`
+    );
```

Comment:
Isn't this always going to throw currently?

Also if you're throwing an error, prefer to put that in the conditional instead of wrapping everything else in it imo.

Reply:
Oh whoops, you're right.

Reply:
Switched the logic around so it throws first on config not found.